src/content/topics/home/using-flags/flag-settings.mdx

```diff
@@ -46,6 +46,8 @@ Here is an image of the settings for all environments on the flag's **Settings**
 
 The flag maintainer is the account member who is primarily responsible for the flag. By default, the maintainer is set to the member who created the flag, but you can assign any member of your team as the maintainer for a particular flag.
 
+Enterprise customers using [teams](/home/teams) can assign a team as the flag's maintainer. This offers the additional benefit of shared accountability and also ensures that when one member leaves the team, the flag still has other owners.
```

Comment:
```suggestion
Enterprise customers using [teams](/home/teams) can assign a team as the flag's maintainer. This offers the additional benefit of shared accountability and also ensures that if one member leaves the team, the flag still has other maintainers.
```

I think it's fine to add a screenshot here if you'd like one @charliepark, as long as you crop it fairly closely. It's not a problem to add a screenshot that's for Enterprise, although we may want to mention that in the caption in addition to here in preceding paragraph. 

Does the `flag-settings-all-environments.png` (line 43) need to be updated based on this feature?

Reply:
The image currently on the page isn't wrong … it's just showing a single member as flag maintainer, which is still a valid state that a flag could be in (either enterprise or not). So we can leave that one as-is, I think.